photutils/psf/photometry.py

```diff
@@ -0,0 +1,427 @@
+                 fitter=LevMarLSQFitter(), niters=3, aperture_radius=None):
+        """
+        Attributes
+        ----------
+        find : callable or instance of any StarFinderBase subclasses
+            ``find`` should be able to identify stars, i.e. compute a rough
+            estimate of the centroids, in a given 2D image.
+            ``find`` receives as input a 2D image an return an
+            `~astropy.table.Table` object which contains columns with names:
+            ``id``, ``xcentroid``, ``ycentroid``, and ``flux``. In which
+            ``id`` is an interger-valued column starting from ``1``,
+            ``xcentroid`` and ``ycentroid`` are center position estimates of
+            the sources and ``flux`` contains flux estimates of the sources.
+            See, e.g., `~photutils.detection.DAOStarFinder`
+        group : callable or instance of any GroupStarsBase subclasses
+            ``group`` should be able to decide whether a given star overlaps
+            with any other and label them as beloging to the same group.
+            ``group`` receives as input an `~astropy.table.Table` object
+            with columns named as ``id``, ``x_0``, ``y_0``, in which ``x_0``
+            and ``y_0`` have the same meaning of ``xcentroid`` and
+            ``ycentroid``. This callable must return an `~astropy.table.Table`
+            with columns ``id``, ``x_0``, ``y_0``, and ``group_id``. The
+            column ``group_id`` should cotain integers starting from ``1``
+            that indicate in which group a given source belongs to.
+            See, e.g., `~photutils.psf.DAOGroup`
+        bkg : callable or instance of any `~photutils.BackgroundBase`
+            subclasses ``bkg`` should be able to compute either a scalar
+            background or a 2D background of a given 2D image.
+            See, e.g., `~photutils.background.MedianBackground`
+        psf : `astropy.modeling.Fittable2DModel` instance
+            PSF or PRF model to fit the data. Could be one of the models in
+            this package like `~photutils.psf.sandbox.DiscretePRF`,
+            `~photutils.psf.IntegratedGaussianPRF`, or any other suitable
+            2D model.
+            This object needs to identify three parameters (position of
+            center in x and y coordinates and the flux) in order to set them
+            to suitable starting values for each fit. The names of these
+            parameters should be given as ``x_0``, ``y_0`` and ``flux``.
+
+            `~photutils.psf.prepare_psf_model` can be used to prepare any 2D
+            model to match this assumption.
+        fitshape : array-like
+            Rectangular shape around the center of a star which will be used
+            to collect the data to do the fitting, e.g. (5, 5) means to take
+            the following relative pixel positions: [-2, -1, 0, 1, 2].
+            Also, each element of ``fitshape`` must be an odd number.
```

Comment:
removed the also in merging.  Thought about also saying it's length-2, but I realized I wanted to make a slight change in that anyway, so can address this in the PR I'm about to submit